tests/src/container-smoke.spec.ts

```diff
@@ -153,9 +162,40 @@ describe('Verification of container creation workflow', async () => {
     const containers = await navigationBar.openContainers();
     const containersDetails = await containers.openContainersDetails(containerToRun);
     await playExpect(containersDetails.heading).toContainText(containerToRun);
-    const containersPage = await containersDetails.deleteContainer(10000);
-    playExpect(containersPage).toBeDefined();
-    playExpect(await containersPage.containerExists(containerToRun)).toBeFalsy();
+    const containersPage = await containersDetails.deleteContainer();
+    await playExpect(containersPage.heading).toBeVisible();
+    await playExpect.poll(async () => await containersPage.containerExists(containerToRun)).toBeFalsy();
     await pdRunner.screenshot('containers-container-deleted.png');
   });
+
+  test('Prune containers', async () => {
+    const navigationBar = new NavigationBar(page);
+
+    for (const container of containerList) {
+      const images = await navigationBar.openImages();
```

Comment:
What functionality? This part only creates some containers, same as previous tests in the file.

Reply:
Exactly, I was thinking that we can extract the logic for creating a container into a helper or utility class so that the code does not need to be duplicated.

Reply:
done